### PR TITLE
Cauldrons!

### PR DESCRIPTION
**Pull request by monster860**
:cl: monster860
rscadd: Adds cauldrons. They are containers of 1000 reagents that can be constructed using 10 metal, and you can insert and remove reagents from them. Sprites by KMC
/:cl:

### CODE REVIEW

**Comment by Partheo** [comment]
wo

**Comment by Partheo** [comment]
Imagine the booms you can make with this

**Comment by Kmc2000** [comment]
credit me u nerd reee

**Comment by ShadowDeath6** [comment]
cool and good

is there a reason you're adding these outside of immense chemical memery, I can't help but think that this will seldom be used since you can't transfer mass amounts of chemicals at once

**Comment by monster860** [comment]
It makes things like mixing mass amounts of synthflesh a lot less painful

**Comment by Time-Green** [comment]
Does it have a build-in heater?

**Comment by X-TheDark** [comment]
Think of all the meth you can make without worrying about all the leftover chems...

**Comment by drderp3635** [comment]
Add brooms when?

**Comment by Ratvar** [comment]
Why not give them inbuilt heaters and remove chemheaters itself?

**Comment by Time-Green** [comment]
I dont think using pyrosium to heat 800 units of meth is a great idea, so please add some sort of weaker heating

File: code/modules/reagents/cauldron.dm [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/obj/structure/reagent_holder
+	name = "cauldron"
+	desc = "A big thing for storing and mixing lots of reagents"
```
Comment (matskuman5) [inline code/modules/reagents/cauldron.dm:3]: THERE'S NO PERIOD REEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE

Comment (Partheo) [reply code/modules/reagents/cauldron.dm:3]: "a big thing"

**Comment by Notamaniac** [comment]
They look more like vats than cauldrons

**Comment by KrabSpider** [comment]
Why 10 metal? Why not have it take 50 sheets and require a few more materials (plasteel)?

**Comment by ghost** [comment]
Sprites aren't suitable imho, make them a witch cauldron,  you can't makr high tech with 10 metal amirite?

**Pull request closed without merging**